XmlFormatter/src/Windows/Settings.cs

```diff
@@ -417,5 +448,67 @@ private void B_EditHotfolder_Click(object sender, EventArgs e)
                 }
             }
         }
+
+        /// <summary>
+        /// Selected log file changed
+        /// </summary>
+        /// <param name="sender">The sender ov the event</param>
+        /// <param name="e">The arguments of the event</param>
+        private void LV_logFiles_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            if (LV_logFiles.SelectedItems.Count == 0)
+            {
+                B_RemoveHotfolder.Enabled = false;
+                return;
+            }
+            string file = LV_logFiles.SelectedItems[0].Tag.ToString();
+            LoadLogFile(file);
+        }
+
+        /// <summary>
+        /// Load the given log file and put it into the rich box
+        /// </summary>
+        /// <param name="file">The file to load</param>
+        private void LoadLogFile(string file)
+        {
+            RTB_loggingText.Text = String.Empty;
+            FileStream stream = new FileStream(file, FileMode.Open, FileAccess.Read, FileShare.Write);
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                RTB_loggingText.Text = reader.ReadToEnd();
+            }
+            B_RemoveHotfolder.Enabled = true;
+        }
+
+        /// <summary>
+        /// Delete the selected log file
+        /// </summary>
+        /// <param name="sender">The sender ov the event</param>
+        /// <param name="e">The arguments of the event</param>
+        private void B_DeleteLog_Click(object sender, EventArgs e)
+        {
+            if (LV_logFiles.SelectedItems.Count == 0)
+            {
+                return;
+            }
+            try
+            {
+                File.Delete(LV_logFiles.SelectedItems[0].Tag.ToString());
+                FillLogFolderView();
+            }
+            catch (Exception)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5095168)